demisto_sdk/commands/content_graph/interface/graph.py

```diff
@@ -131,6 +131,10 @@ def validate_duplicate_ids(self, file_paths: List[str]) -> None:
     def clean_graph(self):
         ...
 
+    @abstractmethod
+    def clear_cache(self):
+        ...
+
```

Comment:
no need IMO, the cache is specific to our neo4j interface implementation

Reply:
But we cannot call methods which only exist in our implementation.
How do you suggest clearing the id_to_obj cache?

Reply:
if it's called from within a method in the neo4j graph interface impl (as suggested above), it will work